_includes/home-page.html

```diff
@@ -189,6 +189,31 @@ <h2><span class="special-text">Context-Aware</span> Policies For Applications</h
 			</div>
 		</div>
 	</section>
+	<section>
+		<div class="container flex">
+			<div class="text ">
+					<img src="{{ site.baseurl }}/assets/images/meshmap-icon.svg"
+						style="max-width:30%;position:relative;float:left;
+						margin-right:20px;margin-bottom: 10px;"
+						alt="Webassembly based Envoy Filters" loading="lazy"/>
+				<h2>Simplify the process of Infrastructure Managment with <span class="special-text">MeshMap</span></h2>
```

Comment:
```suggestion
				<h2>Simplify the process of Infrastructure Managment with <span class="special-text">Meshery Extension</span></h2>
```

Reply:
@karnatisrinivas MeshMap is an extension of Meshery. MeshMap is a plugin of Meshery. It is built separately and loaded dynamically into Meshery UI at runtime